docs/safety.md

```diff
@@ -411,7 +410,7 @@ node and the new Tor node:
 
 ### Prevent data corruption
 
-Many problems while running an `lnd` node can be prevented by avoiding data
+Many problems while running a `lnd` node can be prevented by avoiding data
```

Comment:
since "LND" starts with a vowel sound ("el"), `an` is actually correct here. Same for the other spots and for the "lncli" spots

Reply:
It does indeed. Fixed.